src/components/atoms/CardHeader/CardHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import { styled } from '@mui/material/styles';
+import { ReactNode } from 'react';
+import { Box, Typography } from 'src/components';
+
+export type CardHeaderProps = {
+  title: string;
+  description: string;
+  actions?: ReactNode;
+  children?: ReactNode;
+};
+
+const HeaderStyled = styled(Box)(({ theme: { palette } }) => ({
+  borderRadius: '4px 4px 0px 0px',
+  backgroundColor: palette.uiWhite[500],
+  display: 'flex',
+  alignItems: 'flex-start',
+  justifyContent: 'space-between',
+  padding: '16px 24px',
+  borderBottom: `1px solid ${palette.uiCoolGray[200]}`,
+  minHeight: '48px',
+}));
+
+export const CardHeader = ({
```

Comment:
Also, I would replicate the system we have currently in the dashboard project where the card header is always supposed to be used inside the base card, so I think it makes sense to export it in such a way that it will imported like this
```jsx
<BaseCard>
  <BaseCard.Header title={"title"} />
</BaseCard>
```
See `BaseCard/index.tsx` in the dashboard for reference.

Reply:
+1